packages/jbrowse-plugin-apollo/src/ApolloInternetAccount/model.ts

```diff
@@ -63,6 +63,14 @@ const stateModelFactory = (
         const dec = jwtDecode(token) as JWTPayload
         return dec.roles
       },
+      get userId() {
```

Comment:
This is probably not the most efficient way to do this as I'd assume retrieveToken is relatively expensive operation which would become apparent when many users are in the db.